### PR TITLE
Cleanup trigger client reset function

### PR DESCRIPTION
**Pull request by nirinchev**
## Description

Just a little bit of tweaking - I noticed there was a typo - `useId` vs `userId` which is why we weren't logging the user id (even though the actual deletion was succeeding). I also reshuffled the code a little bit to make it shorter.

### CODE REVIEW

File: Tools/DeployApps/BaasClient.cs [hunk 1]
```diff
@@ -76,25 +76,22 @@ public enum Result
                 };";
 
         private const string TriggerClientResetOnSyncServerFuncSource =
-            @"exports = async function(useId, appId = '') {
```
Comment (LaPeste) [inline Tools/DeployApps/BaasClient.cs:79]: 🤦🏻

File: Tools/DeployApps/BaasClient.cs [hunk 2]
```diff
@@ -76,25 +76,22 @@ public enum Result
...
                   }
-                  deletionResult = await mongodb.db(dbName).collection('clientfiles').deleteMany({ ownerId: useId });
+                  const deletionResult = await mongodb.db(dbName).collection('clientfiles').deleteMany({ ownerId: userId });
                   console.log('Deleted documents: ' + deletionResult.deletedCount);
+
+                  return { status: deletionResult.deletedCount > 0 ? 'success' : 'failure' };
                 } catch(err) {
                   throw 'Deletion failed: ' + err;
                 }
-                if (deletionResult.deletedCount > 0) {
```
Comment (LaPeste) [inline Tools/DeployApps/BaasClient.cs:94]: How do we know if anything was deleted? If we don't care, why?

Comment (nirinchev) [reply Tools/DeployApps/BaasClient.cs:94]: I just moved the return inside the `try` block:

```
return { status: deletionResult.deletedCount > 0 ? 'success' : 'failure' };
```

Comment (LaPeste) [reply Tools/DeployApps/BaasClient.cs:94]: eh... completely blind... Sorry

**Review by LaPeste — APPROVED** [review]

**Pull request merged**